component/host.go

```diff
@@ -54,7 +54,7 @@ type Host interface {
 	// GetExporters returns the map of exporters. Only enabled and created exporters will be returned.
 	// Typically is used to find exporters by type or by full config name. Both cases
 	// can be done by iterating the returned map. There are typically very few exporters
```

Comment:
```suggestion
	// can be done by iterating the returned map. There are typically very few exporters,
```
Compound sentence (Indep, so Indep)